cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -302,6 +303,7 @@ message StreamTracesRequest {
         Seismic3dExtent three_dee_extent = 4; // only valid if the queried object is 3D
         GeometryFilter geometry = 5;
     }
+    repeated LineDescriptor cdp_traces = 8; // Which traces to query within each ensemble. Only valid for pre-stack data
```

Comment:
This is assuming people will want the same parts of each ensemble. If we want to support fetching different parts of each ensemble, the API will have to change much more drastically.